src/main/java/net/minecraftforge/network/NetworkRegistry.java

```diff
@@ -330,10 +330,20 @@ public static void lock() {
          */
         private final String messageContext;
 
+        /**
+         * If the connection should await a response to this packet to continue
+         */
+        private final boolean needsResponse;
+
         public LoginPayload(final FriendlyByteBuf buffer, final ResourceLocation channelName, final String messageContext) {
+            this(buffer, channelName, messageContext, true);
+        }
+
+        public LoginPayload(final FriendlyByteBuf buffer, final ResourceLocation channelName, final String messageContext, final boolean needsResponse) {
```

Comment:
Formatting (braces on new lines)

Reply:
It kind of feels off changing this one method to have braces on newlines if all the other braces are not on a new line either, but whatever